content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Installing External Istiod [experimental]
+description: Install external Istiod and remote config cluster.
+weight: 46
+keywords: [external,istiod,remote,config]
+aliases:
+    - /docs/setup/kubernetes/additional-setup/external-istiod/
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+{{< warning >}}
+This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
+considered `pre-alpha`.
+{{< /warning >}}
+
+## Introduction
+
+This guide walks you through the installation of an external control plane. The {{< gloss >}}external control plane{{< /gloss >}} [deployment model](/docs/ops/deployment/deployment-models/#control-plane-models) enables mesh operators to install and manage mesh control planes on separate external clusters. This deployment model allows a clear separation between mesh operators and mesh admins. Istio mesh operators can run Istio control planes for mesh admins while mesh admins control the configuration of the control plane without worrying about installing or managing it.
+
+## Requirements
+
+### Clusters
+
+This guide requires that you have two Kubernetes clusters with any of the
+supported Kubernetes versions: {{< supported_kubernetes_versions >}}.
+
+The first cluster contains the {{< gloss >}}external control plane{{< /gloss >}} installed
+in the `external-istiod` namespace. This Istio control plane serves as the external control plane for the second cluster.
+An ingress gateway is installed in the `istio-system` namespace. The ingress gateway provides mesh sidecars access to the
```

Comment:
I'd like more information here about how the TLS is working e2e

Reply:
So basically, the sidecars in remote cluster have key/certs to talk to gw when gw has trust-able host with trust-able key and certs.   The connection is terminated at the gateway, then upgraded to simple TLS from gateway to istiod.  Istiod uses JWT to validate the sidecars.  This is not perfect and there is a WI to improve this:  https://github.com/istio/istio/issues/28510  cc @howardjohn @stevenctl